### PR TITLE
UnmintDetails component - redemption status bug fix

### PR DESCRIPTION
**Pull request by kpyszkowski**
There is a bug with `UnmintDetails` component. The **in progress** status occurs to be false positive when the component is rendered with the redemption data initially present.
The solution to the issue was to fallback conditional rendering to the `shouldForceIsProcessCompleted` value which reflects the state when data is initially present and indicates to force the **completed** state.

<details>
<summary>The bug: </summary>

image
</details>

<details>
<summary>Expected behavior:</summary>

image

</details>

### CODE REVIEW

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
Left one comment to look at before the merge

**Review by michalsmiarowski — APPROVED** [review]
LGTM

File: src/pages/tBTC/Bridge/UnmintDetails.tsx [hunk 1]
```diff
@@ -198,9 +198,13 @@ export const UnmintDetails: PageComponent = () => {
...
               display="flex"
-              stepText={shouldDisplaySuccessStep ? "Unminted" : "Unminting"}
+              stepText={
+                shouldDisplaySuccessStep || shouldForceIsProcessCompleted
+                  ? "Unminted"
+                  : "Unminting"
+              }
             >
-              {!shouldDisplaySuccessStep && (
+              {!(shouldDisplaySuccessStep || shouldForceIsProcessCompleted) && (
```
[the remark this answers is not in the document]
Comment (kpyszkowski) [reply src/pages/tBTC/Bridge/UnmintDetails.tsx:209]: Ref commit: 9c44d110f

**Review by michalsmiarowski — CHANGES_REQUESTED** [review]
I think something went wrong while merging `main` into this branch.: https://github.com/threshold-network/token-dashboard/commit/882acb22c4df24d618e45a4a6b1ef7ea4fef9e95#diff-e9cbb0224c4a3d23a6019ba557e0cd568c1ad5e1582ff1e335fb7d99b7a1055dR5

The `REACT_APP_DAPP_DEVELOPMENT_TESTNET_CONTRACTS` should be set to `$DAPP_DEVELOPMENT_TESTNET_CONTRACTS` and not false. Loolks like your `main` branch didn't have the newest changes before the merge and [this change](https://github.com/threshold-network/token-dashboard/pull/662) was not applied

**Review by michalsmiarowski — APPROVED** [review]
LGTM 🔥

**Pull request merged**